util/unifycr/src/unifycr-rm.c

```diff
@@ -169,30 +244,82 @@ static int read_resource_lsf(unifycr_resource_t *resource)
     return 0;
 }
 
+/**
+ * @brief
+ *
+ * @param resource
+ * @param state
+ *
+ * @return
+ */
+static int lsf_launch_daemon(unifycr_resource_t *resource,
+                             unifycr_runstate_t *state)
+{
+    return 0;
+}
+
+static _ucr_resource_manager_t resource_managers[] = {
+    { "none", &none_read_resource, &none_launch_daemon },
+    { "pbs", &pbs_read_resource, &pbs_launch_daemon },
+    { "slurm", &slurm_read_resource, &slurm_launch_daemon },
+    { "lsf", &lsf_read_resource, &lsf_launch_daemon },
+};
+
 int unifycr_read_resource(unifycr_resource_t *resource)
 {
-    int ret = -1;
     char *str = NULL;
 
+    /*
+     * "do not use assignement in if condition"
+     * so, here we use goto instead of if .. else ... any better way?
```

Comment:
Perhaps this construct, otherwise ignore the style warning.

```
char *pbs_jobid = getenv("PBS_JOBID");
char *slurm_jobid = getenv("SLURM_JOBID");
char *lsb_jobid = getenv("LSB_JOBID");

if (pbs_jobid) {
 //
} else if (slurm_jobid) {
 //
} else if (lsb_jobid) {
  //
}
```

Reply:
I have addressed this as you suggested.